stream-chat-android-ui-components/src/main/res/layout/stream_ui_activity_attachment_gallery.xml

```diff
@@ -6,86 +6,155 @@
     android:layout_height="match_parent"
     >
 
-    <ImageView
-        android:id="@+id/closeButton"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/stream_ui_spacing_small"
-        android:background="?selectableItemBackgroundBorderless"
-        android:src="@drawable/stream_ui_ic_close"
-        app:layout_constraintBottom_toBottomOf="@id/toolbarGuideline"
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I don't think we need 2 `ConstraintLayout` nested into a parent `ConstraintLayout` we can do this layout just with one and a `Group` in the bottom bar.

Reply:
I think on this screen the performance is not critical, so we can afford some nested layouts for the sake of readability and ease of support.